<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComponentDialog</class>
 <widget class="QDialog" name="AddComponentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Категория комплектующего</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboTypeName">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Название</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputName">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Гарантия (мес)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputWarranty"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Цена (руб)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputPrice"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>200</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonAddComponent">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCloseDialog">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
         </property>
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCloseDialog</sender>
   <signal>clicked()</signal>
   <receiver>AddComponentDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddComponentDialog</sender>
   <signal>finished(int)</signal>
   <receiver>AddComponentDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
